android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/FxSuggestIngestionScheduler.kt

```diff
@@ -31,9 +31,6 @@ class FxSuggestIngestionScheduler(
     /**
      * Schedules a periodic background task to ingest new suggestions. Does nothing if the task is
      * already scheduled.
-     *
-     * Your main activity's [onCreate][android.app.Activity.onCreate] method should call this
-     * method.
```

Comment:
I dropped this line because this PR calls `startPeriodicIngestion` in `FenixApplication`, not the main activity 😅